sql/moz-fx-data-shared-prod/static/country_codes_v1/data.csv

```diff
@@ -1,253 +1,253 @@
-Japan,JP,JPN,Asia,Eastern Asia,Unspecified,FALSE,FALSE
-Jersey,JE,JEY,Europe,Northern Europe,Unspecified,FALSE,FALSE
-Jordan,JO,JOR,Asia,Western Asia,Unspecified,FALSE,FALSE
-Kazakhstan,KZ,KAZ,Asia,Central Asia,Unspecified,FALSE,FALSE
-Kenya,KE,KEN,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
-Kiribati,KI,KIR,Oceania,Micronesia,Unspecified,FALSE,FALSE
-"Korea, Democratic People's Republic of",KP,PRK,Asia,Eastern Asia,Unspecified,FALSE,FALSE
-"Korea, Republic of",KR,KOR,Asia,Eastern Asia,Unspecified,FALSE,FALSE
-Kuwait,KW,KWT,Asia,Western Asia,Unspecified,FALSE,FALSE
-Kyrgyzstan,KG,KGZ,Asia,Central Asia,Unspecified,FALSE,FALSE
-Lao People's Democratic Republic,LA,LAO,Asia,South-eastern Asia,Unspecified,FALSE,FALSE
-Latvia,LV,LVA,Europe,Northern Europe,Unspecified,FALSE,TRUE
-Lebanon,LB,LBN,Asia,Western Asia,Unspecified,FALSE,FALSE
-Lesotho,LS,LSO,Africa,Sub-Saharan Africa,Southern Africa,FALSE,FALSE
-Liberia,LR,LBR,Africa,Sub-Saharan Africa,Western Africa,FALSE,FALSE
-Libya,LY,LBY,Africa,Northern Africa,Unspecified,FALSE,FALSE
-Liechtenstein,LI,LIE,Europe,Western Europe,Unspecified,FALSE,FALSE
-Lithuania,LT,LTU,Europe,Northern Europe,Unspecified,FALSE,TRUE
-Luxembourg,LU,LUX,Europe,Western Europe,Unspecified,FALSE,TRUE
-Macao,MO,MAC,Asia,Eastern Asia,Unspecified,FALSE,FALSE
-"Macedonia, the Former Yugoslav Republic of",MK,MKD,Europe,Southern Europe,Unspecified,FALSE,FALSE
-Madagascar,MG,MDG,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
-Malawi,MW,MWI,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
-Malaysia,MY,MYS,Asia,South-eastern Asia,Unspecified,FALSE,TRUE
-Maldives,MV,MDV,Asia,Southern Asia,Unspecified,FALSE,FALSE
-Mali,ML,MLI,Africa,Sub-Saharan Africa,Western Africa,FALSE,FALSE
-Malta,MT,MLT,Europe,Southern Europe,Unspecified,FALSE,TRUE
-Marshall Islands,MH,MHL,Oceania,Micronesia,Unspecified,FALSE,FALSE
-Martinique,MQ,MTQ,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
-Mauritania,MR,MRT,Africa,Sub-Saharan Africa,Western Africa,FALSE,FALSE
-Mauritius,MU,MUS,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
-Mayotte,YT,MYT,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
-Mexico,MX,MEX,Americas,Latin America and the Caribbean,Central America,FALSE,FALSE
-"Micronesia, Federated States of",FM,FSM,Oceania,Micronesia,Unspecified,FALSE,FALSE
-"Moldova, Republic of",MD,MDA,Europe,Eastern Europe,Unspecified,FALSE,FALSE
-Monaco,MC,MCO,Europe,Western Europe,Unspecified,FALSE,FALSE
-Mongolia,MN,MNG,Asia,Eastern Asia,Unspecified,FALSE,FALSE
-Montenegro,ME,MNE,Europe,Southern Europe,Unspecified,FALSE,FALSE
-Montserrat,MS,MSR,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
-Morocco,MA,MAR,Africa,Northern Africa,Unspecified,FALSE,FALSE
-Mozambique,MZ,MOZ,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
-Myanmar,MM,MMR,Asia,South-eastern Asia,Unspecified,FALSE,FALSE
-Namibia,NA,NAM,Africa,Sub-Saharan Africa,Southern Africa,FALSE,FALSE
```

Comment:
@m-d-bowerman @sergiosonline I noticed this is failing to deploy after this change.  Is the removal of the code here an accident?

Error:
```
Error while reading data, error message: Required column value is missing.; line_number: 155 byte_offset_to_start_of_line: 11797 column_index: 1 column_name: "code" column_type: STRING
```

Reply:
Total accident. Imputing.